### PR TITLE
Fix Try Opal Issue #11

### PR DESCRIPTION
**Pull request by vais**
Make `print` and `puts` work by providing a custom `$stdout.write_proc`
implementation per @adambeynon suggestion

### CODE REVIEW

File: source/javascripts/try_opal.rb [hunk 1]
```diff
@@ -80,19 +80,18 @@ def eval_code(js_code)
   end
 
   def log_error(err)
-    print_to_output "#{err}\n#{`err.stack`}"
+    print "#{err}\n#{`err.stack`}"
```
Comment (elia) [inline source/javascripts/try_opal.rb]: Shouldn't this be #puts?

**Comment by vais** [comment]
@elia in my testing it does not make a difference in this particular case. Why do you think this should be `puts`?

**Comment by vais** [comment]
Note to self: do not commit at 2am :) I changed `print` to `puts`, but the commit message is backwards, sorry for the confusion guys.

**Comment by vais** [comment]
I will re-do this PR properly.

**Pull request closed without merging**